pkgs/top-level/python-packages.nix

```diff
@@ -186,6 +186,8 @@ in {
 
   actdiag = callPackage ../development/python-modules/actdiag { };
 
+  adafruit-io = callPackage ../development/python-modules/adafruit-io { };
```

Comment:
Looks like unrelated.

Reply:
It is, I was getting a merge conflict error, and added that fixed it...

Reply:
`adafruit-io` was added approx. a month ago. Did you try to rebase your branch on the latest master?

Reply:
No. I will try that.